src/main/kotlin/kotlinx/kover/gradle/plugin/commons/Constants.kt

```diff
@@ -11,5 +11,5 @@ internal object KoverMigrations {
     /** URL of migration guideline from versions 0.5.x to 0.6.0 */
     const val MIGRATION_0_5_TO_0_6 = "https://github.com/Kotlin/kotlinx-kover/blob/v0.6.0/docs/migration-to-0.6.0.md"
```

Comment:
Should this path be changed too?

Reply:
 No, this link is fixed by the label, so the path to the files does not change